src/components/Observations/ObservationsEventsNotification.tsx

```diff
@@ -0,0 +1,85 @@
+import { useMemo } from 'react';
+import { Box } from '@mui/material';
+import { Button, Message } from '@terraware/web-components';
+import strings from 'src/strings';
+import { SEED_COLLECTOR_APP_STORE_LINK, SEED_COLLECTOR_GOOGLE_PLAY_LINK } from 'src/constants';
+import { useLocalization } from 'src/providers';
+import { getLongDate } from 'src/utils/dateFormatter';
+
+export type ObservationEvent = {
+  startDate: string;
+  plantingSiteName: string;
+};
+
+export type ObservationsEventsNotificationProps = {
+  events: ObservationEvent[];
+};
+
+export default function ObservationsEventsNotification({ events }: ObservationsEventsNotificationProps): JSX.Element {
+  const { activeLocale } = useLocalization();
+  const openTab = (url: string) => window.open(url, '_blank');
+
+  const eventStrings = useMemo(() => {
+    const dateEvents: Record<string, string[]> = {};
+
+    events.forEach((event) => {
+      const startDate = getLongDate(event.startDate, activeLocale);
+      dateEvents[startDate] = [...(dateEvents[startDate] ?? []), event.plantingSiteName];
+    });
+
+    return Object.keys(dateEvents).map((key) => {
```

Comment:
Let the keys of `dateEvents` be the `event.startDate`s so that they are sortable here (`Object.keys(dateEvents).sort(...).map(...)`), and call `getLongDate` on them inside the map callback? Unless they are already getting sorted in a way I'm not seeing.

Reply:
That's good feedback, thanks. I don't believe they are being sorted... and they probably should. Will update.